chinese-chicken-curry.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Ginseng Chicken</title>
+    </title>
+  </head>
+  <body>
+    <div><img src="https://souperdiaries.com/wp-content/uploads/2018/03/Chinese-chicken-curry-1.jpg" /></div>
+    <div>
+      <h3>Preparation Time: 15 minutes</h3>
+      <h3>Cook Time: 50 minutes</h3>
+      <h3>Total Time: 70 minutes</h3>
+    </div>
+    <h1>Recipe</h1>
+    <h2>Ingredients</h2>
+    <p>
```

Comment:
I think this p tag might be redundant. A ul tag is a block element already, just like a p tag